pkg/model/const.go

```diff
@@ -180,4 +180,106 @@ const (
 
 	// HelmSecretType indicates the type for secrets created by Helm
 	HelmSecretType = "helm.sh/release.v1"
+
+	// OktetoBinEnvVar defines the okteto binary that should be used
+	OktetoBinEnvVar = "OKTETO_BIN"
+
+	// OktetoSkipCleanupEnvVar defines the okteto binary that should be used
+	OktetoSkipCleanupEnvVar = "OKTETO_SKIP_CLEANUP"
+
+	// OktetoUserEnvVar defines the user is using okteto
+	OktetoUserEnvVar = "OKTETO_USER"
```

Comment:
Is this one the same as `okteto_username`?

Reply:
I think this is only used for defining the user of our integration tests